docs/www/data-archiving.md

```diff
@@ -0,0 +1,94 @@
+When the leadership for the patition changes,
+the reconciliation process cancels all upload jobs that are no longer running on a leader
+and starts new upload jobs for new partitions.
+
+The objects that Redpanda uploads to S3 bucket are:
+
+- Partition manifests
+- Topic manifests
+- Log segments
+
+Partition and topic manifests use *.json extension, and log segments use *.log extension.
+
+- Log paths match the paths in the Redpanda data folder in the format
+    `<random-prefix>/<namespace>/<topic>/<partition-id>_<revision-id>/<logfile-name>.log`.
+    For example, a log segment path looks like `/06160a28/kafka/redpanda-test/0_2/39459-1-v1.log`.
+    The random characters improve performance with S3 data transfers. 
+
+- Partition and topic manifests use prefixes with all symbols set to ‘0’ except the first one.
+    For example, a manifest path looks like `/00000000/meta/kafka/redpanda-test/topic_manifest.json` or `/00000000/meta/kafka/redpanda-test/6_2/manifest.json`.
+
+Redpanda adds the `rp-type` tag to all objects in S3:
+
+- Partition manifests - `rp-type=partition-manifest`
+- Log segments - `rp-type=segment`
+- Topic manifests - `rp-type=topic-manifest`
+
+These tags can be used to selectively delete data from S3.
+Both partition manifests and log segments can be safely deleted from S3 bucket when they get old (for instance, after one year).
+We recommend that you keep topic manifests in order to recover the corresponding topic.
+
+## Configuring data archiving
+
+1. Prepare the cloud storage:
+
+    - Amazon AWS S3
+
+        - Specify expiration rules for the files according to the tags.
+        - Use the IAM service to create a user to access S3.
+            - Grant this user permission to read and create objects.
+            - Copy the access key and secret key to the Redpanda configuration options `archival_storage_s3_access_key` and `archival_storage_s3_secret_key`, respectively.
+
+    - Google Cloud Storage
+
+        - Choose a uniform access control when you create the bucket.
+            - Use bucket level permissions for all objects.
+            - Use a Google managed encryption key.
+        - Create a service user with HMAC keys
+            and copy the keys to the Redpanda configuration options `archival_storage_s3_access_key` and `archival_storage_s3_secret_key`, respectively. 
+
+    > **_Note:_** The secret and access keys are stored in plain text in configuration files.
+
+2. Configure Redpanda:
+
+    - Configure these parameters in the Redpanda configuration:
+
+        | Config parameter name                       | Type         | Descripion                                              |
+        |---------------------------------------------|--------------|---------------------------------------------------------|
+        | archival_storage_enabled                    | boolean      | Enables archival storage feature                        |
```

Comment:
@yougotashovel The parameter names get rendered as _italics_. Not sure how to escape that translation.

Reply:
@bmansheim with back ticks `foo_bar_baz`

Reply:
> @yougotashovel The parameter names get rendered as _italics_. Not sure how to escape that translation.

you're probably looking for backslash escaping `something\_foo`

Reply:
Yep. backticks are fine.